hack/swarmcheck

```diff
@@ -12,9 +15,18 @@ usage() {
 TIMEOUT="${1:-300}"
 
 SECONDS=0
+typeset -i nc
+typeset -i ready
 while true; do
-    nc=$(docker node ls | awk '{ print $1 }' | wc -w)
-    accepted=$(docker node ls --filter membership=accepted | awk '{ print $1 }' | wc -w)
-    [[ $nc -eq $accepted ]] && exit 0
+    # number of nodes + 1 (header)
+    nc=$($amps node ls | wc -l)
+    if [[ $? -ne 0 ]]; then nc=0; fi
```

Comment:
Please leave the commented out version of the previous filter test for `accepted` and a TODO comment to investigate whether this is a docker bug.